### DISCUSSION TITLE
Use minio instead of s3

### DISCUSSION THREAD

**Discussion by zzl221000** [ArroyoSystems/arroyo#128] [Ideas]
Our environment only has a local network, minio cluster, and right now arroyo does not support configuring the endpoint parameter of s3.
Normally, s3-compatible object stores should all work.

**Comment by mwylde** [comment]
Hey @zzl221000 this is now supported in master, and will be part of release 0.6 next week. Object store paths are now specified as a full URL, which can include other endpoints. See #296 for the details.
